translations/fr/try_ruby_150.md

```diff
@@ -0,0 +1,12 @@
+---
+lang:   FR
+title:  Maintenant entre ticket
```

Comment:
This title doesn't sound as good in french as it does in english. Perhaps `Maintenant saisis ticket`?